encrypted-data-model/src/main/java/ch/prevo/open/encrypted/model/MatchForTermination.java

```diff
@@ -78,45 +28,25 @@ public void setTransferInformation(CapitalTransferInformation transferInformatio
         this.transferInformation = transferInformation;
     }
 
-    @Override
```

Comment:
I would add here `toString()` again. I think it could be useful to show the payment information as well.

Reply:
Absolutely, an oversight. Fixed it.